src/features/DisplayControls/PatientHeader/components/PatientMovementModal.jsx

```diff
@@ -0,0 +1,274 @@
+        }
+      } else if (selectedDropDownItem.name === "Undo Discharge") {
+        const response = await undoDischargePatient(
+          visitSummary.dischargeDetails.uuid,
+          selectedDropDownItem.name
+        );
+        if (response.status === 200) {
+          window.location.href = getADTDashboardUrl(
+            patient.uuid,
+            visit,
+            visitSummary.dischargeDetails.uuid
+          );
+        }
+      } else if (selectedDropDownItem.name === "Transfer Patient") {
+        const response = await updatePatientMovement(createPayload());
+        if (response.status === 200) {
+          window.location.href = getADTDashboardUrl(
+            patient.uuid,
+            visit,
+            response.data.encounterUuid
+          );
+        }
+      }
+    }
+  };
+
+  const handleSelectOnChange = (e) => {
+    var item = {};
+    if (e.label === "Admit Patient") {
+      item = {
+        name: "Admit Patient",
+        encounterType: vistEncounterTypes.encounterTypes.ADMISSION,
+      };
+      setShowAdtNotes(true);
+      setSaveEnable(true);
+    } else if (e.label === "Undo Discharge") {
+      item = { name: "Undo Discharge", encounterType: null };
+      setShowAdtNotes(false);
+      setSaveEnable(true);
+    } else if (e.label === "Discharge Patient") {
+      item = {
+        name: "Discharge Patient",
+        encounterType: vistEncounterTypes.encounterTypes.DISCHARGE,
+      };
+      setShowAdtNotes(true);
+      setSaveEnable(true);
+    } else if (e.label === "Transfer Patient") {
+      item = {
+        name: "Transfer Patient",
+        encounterType: vistEncounterTypes.encounterTypes.TRANSFER,
+      };
+      setShowAdtNotes(false);
+      setSaveEnable(true);
+    } else {
+      setSaveEnable(false);
+    }
+    setSelectedDropDownItem(item);
+  };
+
+  const handleNotesOnChange = (e) => {
+    setAdtNotes(e.target.value);
+  };
+
+  return (
+    <>
+      {!isLoading && (
+        <Modal
+          open
+          modalHeading="Patient Movement"
+          primaryButtonText="Save"
+          secondaryButtonText="Cancel"
+          aria-label="Modal content"
+          onRequestClose={() => updatePatientMovementModal(false)}
+          onSecondarySubmit={() => updatePatientMovementModal(false)}
+          onRequestSubmit={() => handleOnSave()}
+          primaryButtonDisabled={!saveEnable}
+        >
+          
+          {dropDown == {} ? (
+            <DropdownSkeleton />
+          ) : (
+            <Dropdown
+              id="patient-movement-dropdown"
+              label="Choose an option"
+              onChange={(e) => handleSelectOnChange(e.selectedItem)}
+              isRequired={true}
+              width={"100%"}
+              items={dropDown}
+              itemToString={(item) => (item ? item.label : "")}
+            />
+          )}
+          &nbsp;&nbsp;
+          {!showAdtNotes && (<div style={{height:"65px"}}></div>)}
```

Comment:
If you are trying to just add some empty spaces you can use a couple of <br/> Tags maybe instead of assigning height/padding

Reply:
I think this can be handled in another way
`<Dropdown/>
couple of <br/> tags as needed
{showAdtNotes && <TextArea/>`

Reply:
Okay sure